atomic_reactor/plugins/post_compress.py

```diff
@@ -52,14 +52,12 @@ def __init__(self, workflow, load_exported_image=False, method='gzip'):
         self.uncompressed_size = 0
         self.source_build = bool(self.workflow.build_result.source_docker_archive)
 
-    def _compress_image_stream(self, stream):
-        outfile = os.path.join(self.workflow.source.workdir,
-                               EXPORTED_COMPRESSED_IMAGE_NAME_TEMPLATE)
+    def _compress_image_stream(self, stream) -> Path:
         if self.method == 'gzip':
-            outfile = outfile.format('gz')
+            outfile = self.workflow.build_dir.exported_compressed_image('gz')
```

Comment:
so this is change which will work only for source builds

for container builds you will need it for all arches